packages/story-editor/src/components/inspector/inspectorProvider.js

```diff
@@ -144,8 +147,13 @@ function InspectorProvider({ inspectorTabs, children }) {
     data: {
       tabs: [
         {
-          id: DESIGN,
-          title: __('Design', 'web-stories'),
+          id: INSERT,
+          title: __('Insert', 'web-stories'),
+          Pane: Library,
```

Comment:
> In Figma, the contents of the panels are adjusted to be wider, so it makes sense to me to make it wider right away already, and then later in a follow-up adjust the design panels.

Yep, the new design is `360px` (like the old Library), I've changed `INSPECTOR_MAX_WIDTH` to `360` and now it looks good (even better than in Figma, because Figma has old design pasted into a wider layout without re-adjusting).

Figma:
![image](https://user-images.githubusercontent.com/21036224/156558107-39a4ede1-ae86-418a-bda2-62846430e499.png)
 
Editor:
![image](https://user-images.githubusercontent.com/21036224/156558205-d907938b-5bc1-4568-b517-e6189929a98f.png)